components/Choice/index.jsx

```diff
@@ -0,0 +1,98 @@
+import React, {Component, PropTypes} from 'react';
+
+import styles from './index.scss';
+
+
+export default class Choice extends Component {
+  static propTypes = {
+    checked: PropTypes.string,
+    choiceStyle: PropTypes.object,
+    disabled: PropTypes.bool,
+    name: PropTypes.string.isRequired,
+    onChange: PropTypes.func,
+    options: PropTypes.object.isRequired
+  };
+
+  static defaultProps = {
+    checked: ''
+  };
+
+  state = {
+    checked: this.props.checked,
+    isHovering: (new Array(Object.keys(this.props.options).length)).fill(false)
+  };
+
```

Comment:
In components that can receive default values we always add componentWillMount () and componentWillReceiveProps () here to make sure the correct value is applied. For instance the value might be given by the props or it might take the value from the route.

Take a look in the CheckboxGroup to see how we did this.. it should be pretty much the same